server/Arcadia.Assistant.Server/Application.cs

```diff
@@ -67,8 +64,27 @@ public virtual void Dispose()
             this.ActorSystem?.Dispose();
             this.container.Dispose();
             this.ActorSystem = null;
-            this.ServerActors = null;
             this.container = null;
         }
+
+        protected virtual void OnStart(ActorSystem actorSystem)
+        {
+        }
+
+        protected virtual void OnStop(ActorSystem actorSystem)
+        {
+        }
+
+        private void CreateRootActors()
```

Comment:
please revert it. what's the point to have everything in one place

Reply:
It also was in one place before - in ActorSystemBuilder. It also created unused collection that contained all top-level actors.